docs/running-bots-guide.md

```diff
@@ -0,0 +1,218 @@
+# Running bots in Zulip
+
+Zulip's features can be extended by the means of bots and integrations.
+
+* **Integrations** are used to connect Zulip with different chat, scheduling and workflow software.
+  If this is what you are looking for, please check out the [integrations guide](
+  http://zulip.readthedocs.io/en/latest/integration-guide.html?highlight=integrations).
+* **Bots**, as a more general concept, intercept and react to messages.
+  If this is what you are looking for, read on!
+
+The purpose of this documentation is to provide you with information about Zulip's
+bot system.
+
+On this page you'll find:
+
+* A step-by-step [tutorial](#how-to-run-a-bot) on how to run a bot.
+* Common [problems](#common-problems) when developing/running bots and their solutions.
+
+Contributions to this guide are very welcome, so if you run into any
+issues following these instructions or come up with any tips or tools
+that help with writing bots, please visit
+[#integrations](https://chat.zulip.org/#narrow/stream/integrations) on the
+[Zulip development community server](https://chat.zulip.org), open an
+issue, or submit a pull request to share your ideas!
+
+## Installing the `zulip_bots` package
+
+The `zulip_bots` package comes with all you need to run a bot.
+
+To install it, run `pip install zulip_bots`.
+
+*Hint: Do you want to install the latest version? Check out [this](
+ writing-bots-guide.html#installing-a-development-version-of-the-zulip-bots-package) guide.*
+
+## How to run a bot
+
+This guide will show you how to run a bot on a running Zulip
+server.  It assumes you want to use one of the existing `zulip_bots/bots`
+bots in your Zulip organization.
```

Comment:
I think `... of the existing zuip_bots/bots bots ...` is a bit confusing. Perhaps change it to `... want to use one of the existing bots (found in `zulip_bots/bots`) in your Zulip organization.`? Or something similar?

Reply:
Definitely sounds better! I'm sure we will find more of these kind of flaws in the original text, so I'll make sure to get this PR through rather quickly, so making changes is not blocked :)